pages/Transfer.qml

```diff
@@ -306,6 +274,54 @@ Rectangle {
           }
       }
 
+      RowLayout{
+          StandardButton {
+              id: resolveButton
+              anchors.left: parent.left
+              width: 80
+              text: qsTr("Resolve") + translationManager.emptyString
+              visible: isValidOpenAliasAddress(addressLine.text)
+              enabled : isValidOpenAliasAddress(addressLine.text)
+              onClicked: {
+                  var result = walletManager.resolveOpenAlias(addressLine.text)
+                  if (result) {
+                      var parts = result.split("|")
+                      if (parts.length == 2) {
+                          var address_ok = walletManager.addressValid(parts[1], appWindow.persistentSettings.testnet)
```

Comment:
`testnet` was replaced by `nettype` when stagenet was introduced. The removed code above was already using it, so I wonder why it's changed here.